src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -33,6 +33,7 @@ const mockConfigurationService = jest.mocked(configurationService);
 describe('RedisCacheService', () => {
   let redisCacheService: RedisCacheService;
   let defaultExpirationTimeInSeconds: number;
+  const keyPrefix = '';
```

Comment:
Is there a reason we don't want to test with the prefix here?

Reply:
I've decided to create a new set of tests to add the prefix: [20dc54f](https://github.com/safe-global/safe-client-gateway/pull/998/commits/20dc54f91051f3a9afceaaffdaa9cdf19f91a11a)

The main reason is because I didn't want to change the validation tests that we have currently to handle the prefix since the application will run without one.

The tests introduced in `redis.cache.service.key-prefix.spec.ts` should cover this though!